source/macos/AdaptiveCards/AdaptiveCards/Views/ACRButton.swift

```diff
@@ -4,10 +4,14 @@ class ACRButton: FlatButton {
     public var backgroundColor: NSColor = .init(red: 0.35216, green: 0.823529412, blue: 1, alpha: 1)
     public var hoverBackgroundColor: NSColor = .linkColor
     public var activeBackgroundColor: NSColor = .gray
-    public var chevron: Bool = true {
+    public var iconImage: String = ""
+    public var iconPos: NSControl.ImagePosition = .imageLeft
+    public var chev: Bool = false {
         didSet {
```

Comment:
Variable names should be verbose enough.
`iconPosition` `showsChevron` `iconImageName`